improver_tests/virtual_temperature/test_VirtualTemperature.py

```diff
@@ -0,0 +1,79 @@
+# (C) Crown Copyright, Met Office. All rights reserved.
+#
+# This file is part of 'IMPROVER' and is released under the BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Unit tests for the VirtualTemperature plugin"""
+
+from unittest.mock import patch, sentinel
+
+import numpy as np
+import pytest
+from iris.cube import Cube
+
+from improver.virtual_temperature import VirtualTemperature
+from improver_tests.utilities.copy_attributes.test_CopyAttributes import HaltExecution
```

Comment:
With the latest changes from a different ticket from  Marcus copy_attributes has been renamed copy_metadata so this now needs to be:
from improver_tests.utilities.copy_metadata.test_CopyMetadata import HaltExecution

Reply:
Done